core-skills/frontend-web-development/rhino-assessee.md

```diff
@@ -0,0 +1,41 @@
+# Rhino
+
+This assessment is composed of 3 discrete parts.
+
+## Semantic HTML
+
+
+We have [prepared a HTML page][rhino-semantic-html-page],
```

Comment:
You can bring your own version if desired.

Reply:
Let's talk about exactly what this means - I've cleaned up the surrounding content a little (https://github.com/madetech/learn/pull/119/commits/f92d877f9fe14a423e57ad5c2b9a7d59c78ab9f0) but I'm still getting my head around what can/should be pre-prepared by an candidate.